src/app.tsx

```diff
@@ -34,28 +34,39 @@ import { NavigatorCardBody } from "./navigator-card-body.jsx";
 import { SidebarPanelDetails } from "./sidebar.jsx";
 import { NavigatorCardHeader } from "./header.jsx";
 import { usePageLocation } from "hooks.js";
-import { fsinfo } from "./fsinfo";
+import { fsinfo, FileInfo } from "./fsinfo";
 
 superuser.reload_page_on_change();
 
+interface Alert {
```

Comment:
Remind me again why we need this?  Isn't patternfly typed?

Reply:
This is something which we made up ourself, a list of "alerts".
```
    const addAlert = (title: string, variant: AlertVariant, key: string) => {
```